### ISSUE TITLE
Running in ubuntu 22.04

### ISSUE THREAD

**Issue by peinfant54** [bluppfisk/coinprice-indicator#94]
Hi, I'm having trouble with a new installation. When I run the app with "./run.sh", I get the next output:

Hello from the pygame community. https://www.pygame.org/contribute.html
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/urllib3/connection.py", line 169, in _new_conn
    conn = connection.create_connection(
  File "/usr/lib/python3/dist-packages/urllib3/util/connection.py", line 96, in create_connection
    raise err
  File "/usr/lib/python3/dist-packages/urllib3/util/connection.py", line 86, in create_connection
    sock.connect(sa)
TimeoutError: timed out

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 699, in urlopen
    httplib_response = self._make_request(
  File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 382, in _make_request
    self._validate_conn(conn)
  File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 1012, in _validate_conn
    conn.connect()
  File "/usr/lib/python3/dist-packages/urllib3/connection.py", line 353, in connect
    conn = self._new_conn()
  File "/usr/lib/python3/dist-packages/urllib3/connection.py", line 174, in _new_conn
    raise ConnectTimeoutError(
urllib3.exceptions.ConnectTimeoutError: (<urllib3.connection.HTTPSConnection object at 0x7fa1401d59f0>, 'Connection to www.mxc.com timed out. (connect timeout=5)')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/pablo/.local/share/Trash/files/coinprice-indicator/lib/python3.10/site-packages/requests/adapters.py", line 489, in send
    resp = conn.urlopen(
  File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 755, in urlopen
    retries = retries.increment(
  File "/usr/lib/python3/dist-packages/urllib3/util/retry.py", line 574, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='www.mxc.com', port=443): Max retries exceeded with url: /open/api/v2/market/symbols (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fa1401d59f0>, 'Connection to www.mxc.com timed out. (connect timeout=5)'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/pablo/.local/share/Trash/files/coinprice-indicator/coin/downloader.py", line 47, in execute
    command.response = get(command.url, timeout=command.timeout)
  File "/home/pablo/.local/share/Trash/files/coinprice-indicator/lib/python3.10/site-packages/requests/api.py", line 73, in get
    return request("get", url, params=params, **kwargs)
  File "/home/pablo/.local/share/Trash/files/coinprice-indicator/lib/python3.10/site-packages/requests/api.py", line 59, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/pablo/.local/share/Trash/files/coinprice-indicator/lib/python3.10/site-packages/requests/sessions.py", line 587, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/pablo/.local/share/Trash/files/coinprice-indicator/lib/python3.10/site-packages/requests/sessions.py", line 701, in send
    r = adapter.send(request, **kwargs)
  File "/home/pablo/.local/share/Trash/files/coinprice-indicator/lib/python3.10/site-packages/requests/adapters.py", line 553, in send
    raise ConnectTimeout(e, request=request)
requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='www.mxc.com', port=443): Max retries exceeded with url: /open/api/v2/market/symbols (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fa1401d59f0>, 'Connection to www.mxc.com timed out. (connect timeout=5)'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/pablo/.local/share/Trash/files/coinprice-indicator/./coin/coin.py", line 371, in <module>
    coin = Coin()
  File "/home/pablo/.local/share/Trash/files/coinprice-indicator/./coin/coin.py", line 55, in __init__
    self._load_assets()
  File "/home/pablo/.local/share/Trash/files/coinprice-indicator/./coin/coin.py", line 128, in _load_assets
    exchange.discover_assets(DownloadService(), lambda *args: None)
  File "/home/pablo/.local/share/Trash/files/coinprice-indicator/coin/exchange.py", line 188, in discover_assets
    downloader.execute(command, cls._handle_discovery_result)
  File "/home/pablo/.local/share/Trash/files/coinprice-indicator/coin/downloader.py", line 51, in execute
    response_handler(None)
  File "/home/pablo/.local/share/Trash/files/coinprice-indicator/coin/exchange.py", line 196, in _handle_discovery_result
    logging.debug("Response from {}: {}".format(command.url, command.error))
AttributeError: 'NoneType' object has no attribute 'url'

What I'm missing?

Thanks

**Comment by bluppfisk** [comment]
Deactivate the mxc plugin. I'm pretty sure that service is down.

**Comment by peinfant54** [comment]
Hi bluppfisk, How can I do that?

Thanks

**Comment by bluppfisk** [comment]
Hello


In user.conf, change mxc: true to mxc: false.


Best regards,

Sander

**Comment by peinfant54** [comment]
Hi bluppfisk.

I did what you said but in my User.conf file, there was no reference to MXC, so I wrote, as you said, "mxc:false", under the plugin section.

After that, when running the app, I got the next error:
ModuleNotFoundError: No module named 'notify2'

But I solved just by installing the notify2 library, with "pip3 install notify2".

After that everything was working perfectly!!!

Thanks for your patience!!
